docs/reference/webhooks-reference.md

```diff
@@ -1,5 +1,173 @@
 # Webhooks
 
-::: up42.webhooks.Webhooks
-    selection:
-        members: {{ funcs_webhooks }}
+The Webhooks class enables creation and modification of custom event notifications with [webhooks](webhooks.md).
+
+```python
+webhook = up42.initialize_webhook(webhook_id="1df1ebb0-78a4-55d9-b806-15d22e391bd3")
+```
+
+A webhook is a method for sending event notifications from one application to another. When something happens in a source system indicated by an event, the webhook transmits an event notification via HTTPS to a specific URL.
```

Comment:
Everything in the overview below the code block is not needed:
- The description is already on the linked page.
- Types of webhooks are already on the linked page.
- The link to DocHub is already on the linked page.

If you check the other reference pages, we don't describe functionalities here, just explain what functionality this class enables.